File: github/modul 6/Bahasa py/PRAK605-2410817320010-RabiahRiskaAmaliah.py
a = int(input())

print("Matriks A:")
matriksa = []
for i in range(a):
    b = list(map(int, input().split()))
    matriksa.append(b)

print("Matriks B:")
matriksb = []
for i in range(a):
    k = list(map(int, input().split()))
    matriksb.append(k) 

matriksab = [[0] * a for _ in range(a)]

for i in range(a):
    for j in range(a):
        for k in range(a):
            matriksab[i][j] += matriksa[i][k] * matriksb[k][j]

print("Matriks AXB:")
for baris in matriksab:
    for k in baris:
        print(k, end=' ')
    print()

File: github/modul 6/Bahasa C/PRAK605-2410817320010-RabiahRiskaAmaliah.c
#include <stdio.h>

int main() {
    int matriksa[10][10], matriksb[10][10], matriksab[10][10];
    int a, i, j, k;
    scanf("%d", &a);

    printf("Matriks A:\n");
    for (i = 0; i < a; i++) {
        for (j = 0; j < a; j++) {
            scanf("%d", &matriksa[i][j]);
        }
    }

    printf("Matriks B:\n");
    for (i = 0; i < a; i++) {
        for (j = 0; j < a; j++) {
            scanf("%d", &matriksb[i][j]);
        }
    }

    for (i = 0; i < a; i++) {
        for (j = 0; j < a; j++) {
            matriksab[i][j] = 0;
        }
    }

    for (i = 0; i < a; i++) {
        for (j = 0; j < a; j++) {
            for (k = 0; k < a; k++) {
                matriksab[i][j] += matriksa[i][k] * matriksb[k][j];
            }
        }
    }

    printf("Matriks AXB:\n");
    for (i = 0; i < a; i++) {
        for (j = 0; j < a; j++) {
            printf("%d ", matriksab[i][j]);
        }
        printf("\n");
    }

    return 0;
}